src/wallet/wallet.h

```diff
@@ -85,6 +85,9 @@ static const bool DEFAULT_WALLET_RBF = false;
 static const bool DEFAULT_WALLETBROADCAST = true;
 static const bool DEFAULT_DISABLE_WALLET = false;
 
+//! Pre-calculated constants for input size estimation in *virtual size*
```

Comment:
nit: how about `Estimated nested input size in *virtual size*, see CalculateNestedKeyhashInputSize`